decorators.rb

```diff
@@ -1,7 +1,8 @@
 require_relative 'person'
 
-class BaseDecorator
+class BaseDecorator < Nameable
```

Comment:
- [ ] <b>REQUIRED:</b> Now `Namable` is in its own file, you now have to also include it here. `require_relative 'nameable'`